### ISSUE TITLE
Quantum doesn't create chunk from lazy import from vue file

### ISSUE THREAD

**Issue by sod** [fuse-box/fuse-box#1360]
Quantum doesn't create chunk from lazy import from vue to vue file. You have to put a javascript file in the middle that has the `import()` code.

Reproduction: https://github.com/sod/fuse-box-vue-lazy-issue

```bash
git clone git@github.com:sod/fuse-box-vue-lazy-issue.git
npm install
node fuse.js production
```

See, only for `LazyDependency2.vue` is a chunk created. Not for `LazyDependency1.vue`. I achieved that by doing the `import('LazyDependency2.vue')` from a javascript file, and importing that instead:

```javascript
// App.vue
console.log('import LazyDependecy1.vue', import('./LazyDependency1.vue')); // < no chunk
console.log('import LazyDependecy2.vue', import('./LazyDependency2WorkaroundImportfile')); // < yes chuck

// LazyDependency2WorkaroundImportfile.js
export default import('./LazyDependency2.vue');
```

**Comment by nchanged** [comment]
@sod thanks for submitting that. Has been fixed in `3.5.0-next.10`

**Comment by sod** [comment]
that was fast :D
